libphal/README.md

```diff
@@ -0,0 +1,10 @@
+PHAL(Power Hardware Abstraction Layer) utility library
+
+	utility library functions for PHAL repository provided APIs.
+This is mainly intended for advanced RAS support for PHAL provided APIs
+in a common place.
+
+To build this package, do the following steps:
+	1. autoreconf -i
+	2. ./configure ${CONFIGURE_FLAGS} --enable-build_libphal
+	3. make 
```

Comment:
This is not true, since you are not building the library separately.  The build instructions should go to top-level README.